source/base/data_out_base.cc

```diff
@@ -6220,7 +6220,8 @@ write_pvd_record (std::ostream &out,
                   const std::vector<std::pair<double,std::string> >  &times_and_names) const
 {
   AssertThrow (out, ExcIO());
-
+   
+  out.precision(12);
```

Comment:
So here's my thought about this: You need higher precision if you have simulations that use time steps that are very different, or if you have many time steps. For example, assume you do 10^7 time steps of size 1. In that case, you lose information when you add up the current time step to the current time, and output it with a small number of digits. In my example, the printed time only changes once every 10 time steps. This leads to all sorts of problems when visualizing things. 

You can avoid that if you output with 8 or 10 or 12 digits. I think 15 is not necessary, though, because I think it is unlikely that you will do 10^15 time steps, and furthermore in that case you get into trouble because *adding time step to time* already incurs round-off on the order of the 15th digit. In other words, the issue then is no longer that we throw away information by printing with too few digits, but that we print information that is no longer correct in the digits that we print. 

In other words, I think 12 is totally sufficient. I can't see a good reason why we *should* print more digits, though of course it also doesn't hurt. (I would just go with 12 consequently.)